packages/backend-app-api/src/services/implementations/httpRouter/httpRouterServiceFactory.ts

```diff
@@ -52,8 +73,14 @@ export const httpRouterServiceFactory = createServiceFactory(
 
       return {
         use(handler: Handler) {
+          router.use(authenticate, handler);
```

Comment:
Beyond the risk to guest users, and static content that isn't set up correctly with cookie auth, what else could this impact?

Reply:
I had issues in the TechDocs with loading CSS files and images does your cookie auth implementation fix that?

Reply:
In my original PR @Rugvip also mentioned the `/app` backend, maybe cookie authentication would be work there as well? 🤔

Reply:
My implementation should now work with TechDocs static content, had to update how we're loading techdocs pages into a multistep process though. Wasn't sure on the /app side, what we should do here.